client/components/MoreButton.tsx

```diff
@@ -0,0 +1,85 @@
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import React, { useState }from 'react';
+import { useSelector } from 'react-redux';
+
+import { RootState } from '../app/store';
+import UpdateZoopDialog from '../components/UpdateZoopDialog';
+
+export interface MoreButtonProps {
+    zoopId: number
+    authorId: number
+    receiverId: number
+    content: string
+}
+
+const MoreButton = ({zoopId, authorId, receiverId, content}: MoreButtonProps) => {
+    const [moreMenuAnchorEl, setMoreMenuAnchorEl] = useState<null | HTMLElement>(null);
+    const [isUpdateDialogOpen, setIsUpdateDialogOpen] = useState(false);
+
+    const currentUser = useSelector((state: RootState) => state.auth.user);
+
```

Comment:
```
if (!currentUser || currentUser.id !== zoop.authorID) {
    return null;
}
```

Reply:
I have it set up so that the ButtonGroup will not render if `!currentUser || currentUser.id !== zoop.authorId`. Should I add this null returning statement to the UpdateZoopButton component that is replacing this MoreButton?